test/yaml/a_mini-multiple-samples.yaml

```diff
@@ -3,12 +3,10 @@ patient_id: 'TWGSAMIN000001'
 # For multi samples, list the BAMs under the corresponding state (normal or tumor).
 input:
   normal:
-    - id: 'TWGSAMIN000001-N001-S01-F'
-      BAM: /hot/resource/SMC-HET/normal/bams/A-mini/n2/output/HG002.N-n2.bam
+    - BAM: /hot/resource/SMC-HET/normal/bams/A-mini/n2/output/HG002.N-n2.bam
+    - BAM: /hot/software/pipeline/pipeline-call-sSNV/Nextflow/development/input/data/A-mini/HG002.N-n2-adjust.bam
```

Comment:
Was it intentional to add a new BAM under `normal` here?

Reply:
Yes, since multiple normals are allowed.